bal/transaction.py

```diff
@@ -228,44 +255,55 @@ def is_valid_address(address):
         return False
     return True
 
+
 def get_new_unspent_tx_outs(new_transactions):
     result = []
     for transaction in new_transactions:
         tx_outs = transaction['tx_outs']
-        tx_objs = [new_unspent_tx_out(transaction['id'], index, tx_out['address'], tx_out['amount']) for index, tx_out in enumerate(tx_outs)]
+        tx_objs = [new_unspent_tx_out(transaction['id'], index, tx_out['address'], tx_out['amount']) for index, 
+                   tx_out in enumerate(tx_outs)]
         result.extend(tx_objs)
 
     return result
 
+
 def get_consumed_tx_outs(new_transactions):
-    return seq(new_transactions)\
-            .map(lambda t : t['tx_ins'])\
-            .reduce(lambda a, b : a + b, [])\
-            .map(lambda tx_in : new_unspent_tx_out(tx_in['tx_out_id'], tx_in['tx_out_index'], '',0))
```

Comment:
I thought it was ok to use "\" for line break in PEP8, is there any other guide that tells the opposite?
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Link you provided is also says that using parenthesis for breaking long lines is the right way. By the way I did not get what you mean by "", I think you mean backslash.

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets  and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.